tests/build/build.go

```diff
@@ -247,6 +286,31 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					return true
 				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to finish")
 			})
+			It("image repo and robot account created successfully", func() {
+
+				component, err := f.AsKubeAdmin.HasController.GetHasComponent(componentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred(), "could not get component %s in the %s namespace", componentName, testNamespace)
+
+				annotations := component.GetAnnotations()
+				imageRepoName, err = build.GetQuayImageName(annotations)
+				Expect(err).ShouldNot(HaveOccurred(), "failed to read image repo name from %+v", annotations)
+
+				imageExist, err := build.DoesImageRepoExistInQuay(imageRepoName)
+				Expect(err).ShouldNot(HaveOccurred(), "failed while checking if image repo exists in quay with error: %+v", err)
+				Expect(imageExist).To(BeTrue(), "quay image does not exists")
+
+				robotAccountName = build.GetRobotAccountName(imageRepoName)
+				robotAccountExist, err := build.DoesRobotAccountExistInQuay(robotAccountName)
+				Expect(err).ShouldNot(HaveOccurred(), "failed while checking if robot account exists in quay with error: %+v")
+				Expect(robotAccountExist).To(BeTrue(), "quay robot account does not exists")
+
+			})
+			It("image tag is updated successfully", func() {
+				//TODO: check if image tag present once below issue is resolved
+				// https://issues.redhat.com/browse/SRVKP-3064
+				// https://github.com/openshift-pipelines/pipeline-service/pull/632
```

Comment:
should a follow-up jira ticket be created to update this test spec once https://github.com/redhat-appstudio/infra-deployments/pull/1847 is merged?